.ci/Jenkinsfile

```
@@ -244,33 +253,93 @@ pipeline {
   }
 }
 
-def checkTestSuite(Map parallelTasks = [:], Map item = [:]) {
-    def suite = item.suite
-    def platforms = item.platforms
+def sshexec(connection, cmd){
+  sh "ssh -tt -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${env.WORKSPACE}/${env.BASE_DIR}/e2essh ${connection} -- '${cmd}'"
+}
+
+def scpr(connection, remote_src, local_dst){
+  sh "scp -r -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${env.WORKSPACE}/${env.BASE_DIR}/e2essh ${connection}:${remote_src} ${local_dst}"
+}
 
-    item.scenarios.each { scenario ->
-      def name = scenario.name
-      def platformsValue = platforms
-      def scenarioPlatforms = scenario.platforms
-      if (scenarioPlatforms?.size() > 0) {
-        // scenario platforms take precedence over suite platforms, overriding them
-        platformsValue = scenarioPlatforms
+/*
+ * Runs python in venv
+ */
+def pyrun(args){
```

Wdyt? Because we are passing command AND args 🤔 
```suggestion
def pyrun(command){
```